tsconfig.json

```diff
@@ -1,41 +1,9 @@
 {
   "extends": "./tsconfig.build.json",
+  "include": ["./packages"],
   "compilerOptions": {
     "baseUrl": ".",
     "strict": true,
-    "noImplicitAny": true,
-    "paths": {
-      //"@dendronhq/*": ["packages/*/src"],
-      "@dendronhq/api-server": [
-        "packages/api-server/src"
-      ],
-      "@dendronhq/common-all": [
-        "packages/common-all/src"
-      ],
-      "@dendronhq/unified": [
-        "packages/unified/src"
-      ],
-      "@dendronhq/common-server": [
-        "packages/common-server/src"
-      ],
-      "@dendronhq/common-test-utils": [
-        "packages/common-test-utils/src"
-      ],
-      "@dendronhq/engine-test-utils": [
-        "packages/engine-test-utils/src"
-      ],
-      "@dendronhq/dendron-cli": [
-        "packages/dendron-cli/src"
-      ],
-      "@dendronhq/engine-server": [
-        "packages/engine-server/src"
-      ],
-      "@dendronhq/plugin-core": [
-        "packages/plugin-core/src"
-      ],
-      "@dendronhq/pods-core": [
-        "packages/pods-core/src"
-      ]
-    }
+    "noImplicitAny": true
```

Comment:
the paths were necessary for vscode to be able to generate import statements. is that no longer the case?

Reply:
you are right we need the paths that imports are resolved correctly in the editor. its the same in vscode and neovim.

Reply:
https://github.com/dendronhq/dendron/pull/3639/commits/599edd31a48c1aca19245c8532fa880e4ed84f14